src/workflows-app/SubmissionDetails.test.js

```diff
@@ -553,4 +553,140 @@ describe('Submission Details page', () => {
 
     expect(screen.getByText('Some workflow statuses are not up to date. Refreshing the page may update more statuses.')).toBeInTheDocument();
   });
+
+  it('should display inputs on the Inputs tab', async () => {
+    const user = userEvent.setup();
```

Comment:
We've been trying to adopt the convention of labeling the "Arrange", "Act", and "Assert" sections of each test with comments.

```suggestion
    // Arrange
    const user = userEvent.setup();
```